screens/gallery/galleryComponents/AlbumList.js

```diff
@@ -43,8 +43,8 @@ class AlbumList extends Component {
 const mapStateToProps = state => {
     return {
         imagesLoading: state.gallery.imagesLoading,
-        gallery: state.gallery.gallery,
-        geotaggedImages: state.gallery.geotaggedImages
+        geotaggedImages: state.gallery.geotaggedImages,
+        images: state.images.images
```

Comment:
what about `state.images.imagesArray`

Reply:
the name of images array in the state is `images`

Reply:
I know that, but I always find `images.images` confusing. What do you think about `images.imagesArray`?